src/main/java/net/sf/jabref/logic/bibtexkeypattern/BibtexKeyPatternUtil.java

```diff
@@ -692,7 +665,7 @@ else if (val.matches("edtr\\d+")) {
                 String ss = entry.getFieldOrAlias(FieldName.YEAR).orElse("");
                 if (ss.isEmpty()) {
                     return ss;
-                } else if (ss.startsWith("in") || ss.startsWith("sub")) {
+                } else if (ss.startsWith("in") || ss.startsWith("sub")) { // TODO: Year starting with in or sub? Why?
```

Comment:
Ahh, I think I understand it now: `IP` = in press/preparation, `in` = in press/preparation, and `sub` = submitted. Better to use the note field for that in general I guess, but maybe worthwhile keeping for compatibility. (One may even add support `under review` leading to either `IP` or `UR`.)